docs/vm-images.md

```diff
@@ -1,11 +1,16 @@
 # Supported and Tested VM Images
 
-* [HPC CentOS 7](#hpc-centos-7)
-* [HPC Rocky Linux 8](#hpc-rocky-linux-8)
-* [Debian 11](#debian-11)
-* [Ubuntu 20.04 LTS](#ubuntu-2004-lts)
-* [Windows](#windows)
+* [General Information](#general-information)
```

Comment:
This page was initially created to call out what images we officially support. I don't want to lose visibility of that. I propose the following title changes:

Supported and Tested VM Images -> VM Images  
General Information -> Specify an Image in a Blueprint  
VM Image Specifics -> HPC Toolkit Supported Images  

Indent `other images` and `slurm on gcp custom images` to be under `HPC Toolkit Supported Images`.

Reply:
Done.